todo_app/app.py

```diff
@@ -10,12 +12,29 @@
 
 @app.route('/', methods=['GET'])
 def index():
-    return render_template('index.html', items = get_items())
+
+    statuses = get_statuses()
+    allItems = get_items()
+
+    return render_template('index.html', statuses = list(statuses), items = list(allItems))
 
 
 @app.route('/add_item', methods=['POST'])
 def addItem():
     title = request.form.get('title')
     add_item(title)
-    return render_template('index.html', items = get_items())
+    statuses = get_statuses()
+    allItems = get_items()
+    return render_template('index.html', statuses = list(statuses), items = list(allItems))
```

Comment:
Minor, but redirecting in response to a successful POST form submission plays slightly nicer with browser behaviour (e.g. refreshing), and means you don't need to duplicate code.

```suggestion
    return redirect('/', 303)
```

Reply:
done